packages/fxa-content-server/server/config/surveys.json

```diff
@@ -4,6 +4,7 @@
     "conditions": {
       "browser": "firefox"
     },
+    "rate": 0.05,
```

Comment:
This value is from https://jira.mozilla.com/browse/FXA-1998.  If you wish to see the survey locally, then change it to 0.99 or 1 or something high enough.